build/android/build.sh

```diff
@@ -56,7 +56,7 @@ fi
 # Continuous builds will build everything
 ANDROID_ABIS=
 if [[ "$TARGET" == "presubmit" ]]; then
-  ANDROID_ABIS="-q arm64-v8a,x86"
```

Comment:
Is this to further reduce build times? Can you update the comment above?

Reply:
It's just temporary, I'm trying to figure out a fix.